CMakeLists.txt

```diff
@@ -318,6 +318,7 @@ if(MSVC)
   endif()
   include_directories(SYSTEM src/platform/msc)
 else()
+  include(TestCXXAcceptsFlag)
   set(ARCH native CACHE STRING "CPU to build for: -march value or 'default' to not pass -march at all")
   message(STATUS "Building on ${CMAKE_SYSTEM_PROCESSOR} for ${ARCH}")
   if(ARCH STREQUAL "default")
```

Comment:
This is the section that sets -march (see next few lines). I vote for not adding another place which sets up -march. It would be better to move this section down, and amend this section to append the feature suffix as a variable.